src/components/ChildComponent.js

```diff
@@ -0,0 +1,34 @@
+function ChildComponent() {
+  return (
+    <div className="calculator">
+      <input className="inputContainer" type="text" id="myInput" name="name" pattern="[0-9+\-*/(). ]+" />
+      <div className="buttons">
+        <input className="calculatorButton mathButton" type="button" value="AC" />
+        <input className="calculatorButton mathButton" type="button" value="+/-" />
+        <input className="calculatorButton mathButton" type="button" value="%" />
+        <input className="calculatorButton mathButtonOrange" type="button" value="÷" />
+        <br />
+        <input className="calculatorButton" type="button" value="7" />
+        <input className="calculatorButton" type="button" value="8" />
+        <input className="calculatorButton" type="button" value="9" />
+        <input className="calculatorButton mathButtonOrange" type="button" value="x" />
+        <br />
+        <input className="calculatorButton" type="button" value="4" />
+        <input className="calculatorButton" type="button" value="5" />
+        <input className="calculatorButton" type="button" value="6" />
+        <input className="calculatorButton mathButtonOrange" type="button" value="-" />
+        <br />
+        <input className="calculatorButton" type="button" value="1" />
+        <input className="calculatorButton" type="button" value="2" />
+        <input className="calculatorButton" type="button" value="3" />
+        <input className="calculatorButton mathButtonOrange" type="button" value="+" />
+        <br />
+        <input className="calculatorButton zero" type="button" value="0" />
+        <input className="calculatorButton" type="button" value="." />
+        <input className="calculatorButton mathButtonOrange" type="button" value="=" />
+      </div>
+    </div>
```

Comment:
- [OPTIONAL BUT RECOMMENDED] Please, kindly ensure that your code is DRY, by not repeating this all along. I think it will be more ideal that your code is dry by trying to put all this repetition in an array, then looping through them, something like

```
const buttonKeys = ['AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '=']

<div>
 {buttonKeys.map((buttonkey) => {
                return <button type='button' key={buttonkey.id} onClick={() => handleClick(buttonkey)}>
                      {buttonkey}
                </button>
            })}
</div>
```

See how that long line of code, becomes that short, whoops 😄.
Then the styling can be done using css selectors, you can do that with logic also, but with CSS selectors is less painful 🤣.

- This is a requirement, it has just been marked as optional because the previous reviewer failed to mention it.

Reply:
Thank you, I will refactor the code now before the next milestone.